charts/keeper/templates/role-cronjob.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
As we are allowing to customize the image, some might need a different RBAC control. I would make this optional, but enabled by default.